python/ray/experimental/state.py

```diff
@@ -386,3 +386,118 @@ def task_profiles(self):
           if "function_name" in event[3]:
             task_info[task_id]["function_name"] = event[3]["function_name"]
     return task_info
+
+  def dump_catapult_trace(self, path):
+    task_info = self.task_profiles()
+    workers = self.workers()
+    tasks = self.task_table()
+    start_time = None
+    for info in task_info.values():
+      task_start = min(self.get_times(info))
+      if not start_time or task_start < start_time:
+        start_time = task_start
+
+    def micros(ts):
+      return int(1e6 * (ts - start_time))
+
+    with open(path, "w") as f:
+      f.write("[\n")
```

Comment:
Is the file we are writing one big JSON file? If so, can we just call `json.dumps` on a single Python object instead of manually writing brackets and commas?